core/src/Microsoft.Mcp.Core/AssemblyInfo.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Mcp.Core.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Mcp.Core.LiveTests")]
+[assembly: InternalsVisibleTo("Microsoft.Mcp.Tests")]
+[assembly: InternalsVisibleTo("AzureMcp.Core.UnitTests")]
```

Comment:
We can't be expected to add every consumers' tests projects to this core project's InternalsVisibleTo.   If some class or member needs to be exposed to a consumer's tests, that class or member should probably just be made public